src/views/landing/NewPlayerHome/NewPlayerHome.vue

```diff
@@ -115,12 +114,9 @@
     }
 
     get newPlayerRoadMap() {
-      return (
-        this.pageData.achievement_roadmap
-        && this.pageData.achievement_roadmap.filter(
-          (p: { key: string }) => (p.key as string) === 'ten_tools',
-        )
-      );
+      return (this.pageData?.achievement_roadmap || []).filter(
+        (p: { key: string }) => p.key === 'ten_tools',
+      ) as Array<AchievementItem>;
```

Comment:
What is the difference between AchievementItem and Achievement? Can the type of `achievement_roadmap` be changed to `AchievementItem[]`?